src/components/elements/query-image.tsx

```diff
@@ -1,11 +1,11 @@
 import React from 'react'
 import styled from 'styled-components'
-import { GatsbyImage, getImage } from 'gatsby-plugin-image'
+import { GatsbyImage, getImage, ImageDataLike } from 'gatsby-plugin-image'
 
 type QueryImageProps = {
     alt: string
     className?: string
-    data: Record<string, unknown>
+    data: ImageDataLike
```

Comment:
Great that you found this! 🎉 👍

Reply:
@sean-binary  I remember suggesting this on your PR 😢  But it seems that you like @mitra-fs  more than me 🤣

Reply:
Really?? @prince-deriv I don't remember this 😂  My bad man!

Reply:
![the-simpsons-homer-simpson](https://user-images.githubusercontent.com/64970259/145957678-961d474d-4ef8-4564-b9a9-e64ad578d3ab.gif)